oapth/src/integration_tests/db/pg.rs

```diff
@@ -10,6 +10,7 @@ pub async fn clean_drops_all_objs<B>(
   c.back_end.execute("CREATE SCHEMA bar").await.unwrap();
   crate::integration_tests::create_foo_table(c, "bar.").await;
   assert_eq!(c.back_end.tables("bar").await.unwrap().len(), 1);
+  c.back_end.execute("CREATE VIEW view AS SELECT * FROM bar.foo WHERE id = 1;").await.unwrap();
```

Comment:
All custom schemas are dropped first and within the dropping procedure, this view is referring an unknown table from a unknown schema

```suggestion
  c.back_end.execute("CREATE VIEW view AS SELECT * FROM foo WHERE id = 1;").await.unwrap();
```

I was able to drop the view by applying the cited change and moving

```
  
  for view in views(back_end, "public").await? {
    dbg!(&view);
    buffer.write_fmt(format_args!("DROP VIEW {} CASCADE;", view))?;
  }
```

Right above

```
  for table in back_end.tables("public").await? {
    buffer.write_fmt(format_args!("DROP TABLE {} CASCADE;", table))?;
  }
```

Reply:
@c410-f3r  attempting this